pages/learn/develop/integrations.md

```diff
@@ -5,11 +5,11 @@ excerpt: Guides to integrate {{ $names.company.lower }} with a variety of IoT pl
 
 # Integrate with 3rd parties
 
-{{ $names.company.upper }} can be integrated with external 3rd party services, such as IoT messaging or data analytics platforms. Here are some guides to help you get started:
+{{ $names.company.upper }} can be integrated with external 3rd party services such as IoT messaging or data analytics platforms. Below are some tools to help you get started.
 
-### Guides
+### Tools
+
```

Comment:
Why redirect by default to AWS?
Why mention lambda specifically? Are you trying to provide customers with a reference or we lean more closely to AWS?

Reply:
No longer applicable.